src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Migration/v0.04/01-setup-tables.sql

```diff
@@ -0,0 +1,50 @@
+-- Enum: delegation.delegationChangeType
+DO $$ BEGIN
+    CREATE TYPE delegation.delegationChangeType AS ENUM ('grant', 'revoke', 'revokelast');
+EXCEPTION
+    WHEN duplicate_object THEN null;
+END $$;
+
+-- Table: drop/create delegation.delegationChanges table both to drop old content and change from isDeleted to delegationChangeType
+DROP TABLE IF EXISTS delegation.delegationChanges CASCADE;
+CREATE TABLE IF NOT EXISTS delegation.delegationChanges
+(
+  delegationChangeId bigint GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  delegationChangeType delegation.delegationChangeType NOT NULL,
+  altinnAppId character varying COLLATE pg_catalog."default" NOT NULL,
+  offeredByPartyId integer NOT NULL,
+  coveredByPartyId integer,
+  coveredByUserId integer,
+  performedByUserId integer NOT NULL,
+  blobStoragePolicyPath character varying COLLATE pg_catalog."default" NOT NULL,
+  blobStorageVersionId character varying COLLATE pg_catalog."default",
+  created timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP
+)
+TABLESPACE pg_default;
+
+-- Set start value of delegationChangeId to 100K to allow some room for manual debugging without collision
+ALTER SEQUENCE delegation.delegationchanges_delegationchangeid_seq RESTART WITH 100000 INCREMENT BY 1;
```

Comment:
Would have been nice if this only executed in non-Dev environments. So Dev could test with <100K values against AT environments

Reply:
Will have to be reset manually in dev before testing against AT for now, since we have no env-variables or specific server naming to separate execution based on env.